apl/scalar-functions/mathematical-functions/max-of.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: max_of
+description: 'This page explains how to use the max_of function in APL.'
+---
+
+Use the `max_of` function in APL (Axiom Processing Language) to return the maximum value from a list of scalar expressions. You can use it when you need to compute the maximum of a fixed set of values within each row, rather than across rows like with aggregation functions. It is especially useful when the values you want to compare come from different columns or are dynamically calculated within the same row.
+
+Use `max_of` when you want to:
+
+- Compare multiple fields in a single event to determine the highest value.
+- Perform element-wise maximum calculations in datasets where values are spread across columns.
+- Evaluate conditional values and select the highest one on a per-row basis.
```

Comment:
Ensure a minimum value. So if I do `max_of(myval, 0)` that means what is returned is always greater than 0.